### PR TITLE
[D1] Add wrangler d1 insights

### PR DESCRIPTION
**Pull request by vy-ton**
- Hold off on adding `wrangler d1 insights` to https://developers.cloudflare.com/workers/wrangler/commands/#d1 until CLI command and output is finalized
- Mention `wrangler d1 insights` on https://developers.cloudflare.com/d1/observability/metrics-analytics/ with examples to increase awareness

### CODE REVIEW

**Comment by vy-ton** [comment]
@rozenmd mind reviewing?

File: content/d1/observability/metrics-analytics.md [hunk 1]
```diff
@@ -134,3 +138,29 @@ query {
 	}
 }
 ```
+
+## Query insights
+D1 exposes metrics that let you understand and debug query performance. These metrics can be accessed via GraphQL's `d1QueriesAdaptiveGroups` or `wrangler d1 insights`command.
+
+- Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), which can contain sensitive information
```
Comment (Maddy-Cloudflare) [inline content/d1/observability/metrics-analytics.md]:
```suggestion
- Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), which can contain sensitive information.
```

File: content/d1/observability/metrics-analytics.md [hunk 2]
```diff
@@ -134,3 +138,29 @@ query {
...
+- Experimental wrangler command options and output can change, run `wrangler d1 insights --help` to view current command
```
Comment (Maddy-Cloudflare) [inline content/d1/observability/metrics-analytics.md]:
```suggestion
- Experimental wrangler command options and output can change. Run `wrangler d1 insights --help` to view the current command.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/d1/observability/metrics-analytics.md [hunk 3]
```diff
@@ -10,6 +10,17 @@ D1 exposes database analytics that allow you to inspect query volume, query late
 
 The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically](#query-via-the-graphql-api) via GraphQL or HTTP client.
 
+## View via the dashboard
```
Comment (rozenmd) [inline content/d1/observability/metrics-analytics.md]:
```suggestion
## View metrics in the dashboard
```

File: content/d1/observability/metrics-analytics.md (hunk 1 above)
Comment (rozenmd) [inline content/d1/observability/metrics-analytics.md]:
```suggestion
- Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), to make it easier to analyze metrics across queries.
```

**Pull request merged**